src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        return userService.findByEmail(email) != null
```

Comment:
it will be a good move to declare an separate variable for `userService.findByEmail(email)` as you use it twice.

Reply:
i made a new commit under. Declare new variable user